docs/01-overview/02-main-areas/service-management/smgt-16-hb-addons-validation.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Addons validation
+---
+
+Checker is a tool that validates addons in the [`addons`](https://github.com/kyma-project/addons) repository on every pull request. It checks whether all [required](./smgt-11-hb-create-addons.md) fields are set in your addons.
+
+The Checker also triggers the [`helm lint`](https://v2.helm.sh/docs/helm/#helm-lint) command using Helm CLI 2.16.1, which checks your addons' charts.
+Run the Checker locally to test if your addons are valid:
```

Comment:
could this be useful in troubleshooting too?

Reply:
I'm not sure this should go to the troubleshooting 🤔 We don't have any specific problem to solve here. It's just a description of a tool that runs on every PR of the `addons` repo and checks if the addons are created correctly.